lib/converter.rb

```diff
@@ -28,7 +30,8 @@ def convert_roman_to_arabic(input)
                 result += current_value
                 i += 1
             end
+            converted_value = { :value => result }
         end
-        result
+        JSON.generate(converted_value)
```

Comment:
Why does the `convert_roman_to_arabic` need to change its interface in order to accommodate the API server? I feel that the server should be the thing that converts responses to JSON.

Reply:
This is what I was sneakily getting at with my goofy comment. Consider the difference between some core logic and how the result of that logic is _presented_ to some user.